redhat/frr.spec.in

```diff
@@ -223,12 +233,17 @@ Contributed/3rd party tools which may be of use with frr.
 
 %package pythontools
 Summary: python tools for frr
-%if 0%{?rhel} && 0%{?rhel} > 7
-BuildRequires: python2
-Requires: python2-ipaddress
+%if 0%{?rhel} && 0%{?rhel} < 7
+#python27 is available from ius community repo for RedHat/CentOS 6
+BuildRequires:  python27
+Requires:  python27-ipaddress
+%else
+%if %{use_python2}
+BuildRequires:  python2
+Requires:  python2-ipaddress
 %else
-BuildRequires: python
-Requires: python-ipaddress
```

Comment:
afaik `ipaddress` is not listed as an rpm package in default repositories, isn't it needed by `frr-pythontools`?

Reply:
there is no python-ipaddress in python3. It is/was listed on older repos
But I'll verify again to make sure the reload script works as intended and will follow up on this comment

Reply:
Verified.
`ipaddress` is part of the Python3 standard library and included in python3. 
`python-ipaddress` is the backported package to provide the same to python2
(There is also an older lib called `ipaddr` which was superseded by `ipaddress`)

As such, the package is only needed for python2 and no extra install with python3

On a sidenote, the `frr-reload.py` script is written to work with python2, python3 and can use ipaddress or ipaddr:
```
try:
    from ipaddress import IPv6Address, ip_network
except ImportError:
    from ipaddr import IPv6Address, IPNetwork
```

So the only real change needed was the shebang at the beginning.

Reply:
awesome 👏